static/src/javascripts/projects/common/modules/commercial/membership-engagement-banner.spec.js

```diff
@@ -324,25 +315,14 @@ describe('Membership engagement banner', () => {
                 );
             }));
 
-        it('paypal and credit card image', () =>
-            membershipEngagementBanner.canShow().then(canShow => {
-                expect(canShow).toBe(true);
-
-                membershipEngagementBanner.show();
-
-                expect(FakeMessage.prototype.show.mock.calls[0][0]).toMatch(
-                    /fake-paypal-and-credit-card-image/
-                );
-            }));
-
         it('colour class', () =>
             membershipEngagementBanner.canShow().then(canShow => {
                 expect(canShow).toBe(true);
 
                 membershipEngagementBanner.show();
 
                 expect(FakeMessage.prototype.show.mock.calls[0][0]).toMatch(
-                    /support-the-guardian-banner/
+                    /engagement-banner/
```

Comment:
why the rename of `support-the-guardian-banner` to `engagement-banner`?

Reply:
In a previous PR I changed the naming in the CSS to be `support-the-guardian-banner`. When I tried to implement that change across the board I realised that it wasn't a great idea - there was so much to change, including some things we really wouldn't want to (e.g. ACQUISITIONS_ENGAGEMENT_BANNER in the data lake), and the phrase "engagement banner" is so widely understood that we might as well stick with it.

So in the PR I changed the few "support the guardian banner" bits to "engagement banner"